src/final1/subtests/InvalidRecommendTerms.java

```diff
@@ -17,6 +18,10 @@
 public class InvalidRecommendTerms extends RecommendationSubtest {
 	private String[] commands;
 
+	public InvalidRecommendTerms() {
+		setExpectedSystemStatus(SystemExitStatus.EXACTLY.status(1));

```

Comment:
Of course. I really should go to bed. Could you sneak that into #65? There's already so much in there, this won't do too much harm.